scripts/firedrake-install

```diff
@@ -1006,6 +1006,114 @@ def build_and_install_slepc():
         log.info("No need to rebuild slepc4py")
 
 
+def build_and_install_vtk():
```

Comment:
I'm a bit nervous about having this whole package definition in firedrake-install. Would it be better to have a git repo containing a Python package which does this instead.